### ISSUE TITLE
WithAzureOpenAITextGeneration but for summarizaion only

### ISSUE THREAD

**Issue by KSemenenko** [microsoft/kernel-memory#133]
I think we can use one model for summarization (when we build data for vector db) like gpt3.5 and anoter like gpt4 for actual text writeng

**Comment by dluc** [comment]
That's correct. Currently the service uses the same model for questions and summarization. You can use a different handler for summarization though, with you custom settings. Plugging in custom handlers is the best way to customize pipelines.

**Comment by dluc** [comment]
note to team: we need to provide concrete examples showing how to use custom handlers and how to use custom models

**Comment by KSemenenko** [comment]
Yes please 🙏
